README.md

```diff
@@ -69,13 +69,19 @@ Please contact mehdi@umich.edu if you have any questions.
 
 #Generators
```

Comment:
Add space between "#" and "Generators" for it to be considered as a section in the markdown language.
`# Generators`